core/mono-samples/Android/AndroidSampleApp.csproj

```diff
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="BuildApp">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier>android-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Runtime.Android.Sample.Mono" Version="6.0.0-*" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <Target Name="BuildNativeLib" BeforeTargets="Publish">
+    <Exec Command="cmake -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK_ROOT/build/cmake/android.toolchain.cmake -DANDROID_ABI=x86_64 -DANDROID_STL=none -DANDROID_NATIVE_API_LEVEL=21 -B monodroid -DCMAKE_BUILD_TYPE=MinSizeRel -DCMAKE_C_FLAGS=%22-s -Wno-unused-command-line-argument%22" />
+    <Exec Command="cmake --build monodroid --config MinSizeRel" />
+
+    <ItemGroup>
+      <NativeLibFiles Include="$(MSBuildThisFileDirectory)monodroid/libnative-lib.so" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(NativeLibFiles)" DestinationFolder="$(ProjectDir)$(PublishDir)" />
+
+    <RemoveDir Directories="$(MSBuildThisFileDirectory)monodroid" />
+  </Target>
+
+  <UsingTask TaskName="AndroidAppBuilderTask"
+             AssemblyFile="$(PkgMicrosoft_NET_Runtime_Android_Sample_Mono)\tools\net6.0\AndroidAppBuilder.dll"/>
+
+  <Target Name="BuildApp" AfterTargets="Publish">
+    <PropertyGroup>
+      <OutputPath>$(ProjectDir)$(PublishDir)</OutputPath>
+      <ApkDir>$(OutputPath)apk\</ApkDir>
+      <AdbTool>$(ANDROID_SDK_ROOT)\platform-tools\adb</AdbTool>
+      <RunActivity>true</RunActivity>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <AndroidRuntimePack Include="@(ResolvedRuntimePack)" Exclude="Microsoft.AspNetCore.App.Runtime.linux-x64" />
+    </ItemGroup>
+
+    <RemoveDir Directories="$(ApkDir)" />
+
+    <AndroidAppBuilderTask
+      RuntimeIdentifier="android-x64"
+      SourceDir="$(OutputPath)"
+      MonoRuntimeHeaders="%(AndroidRuntimePack.PackageDirectory)\runtimes\android-x64\native\include\mono-2.0"
+      MainLibraryFileName="$(AssemblyName).dll"
+      ProjectName="HelloAndroid"
+      OutputDir="$(ApkDir)"
+      StripDebugSymbols="True"
+      KeyStorePath="/Users/mdhwang/.android/"
+      NativeMainSource="$(MSBuildThisFileDirectory)MainActivity.java">
+      <Output TaskParameter="ApkBundlePath" PropertyName="ApkBundlePath" />
+      <Output TaskParameter="ApkPackageId" PropertyName="ApkPackageId" />
+    </AndroidAppBuilderTask>
+
+    <Exec Command="$(AdbTool) kill-server"/>
+    <Exec Command="$(AdbTool) start-server"/>
+    <Exec Command="$(AdbTool) logcat -c" ContinueOnError="WarnAndContinue" />
+    <Exec Command="$(AdbTool) install $(ApkDir)/bin/HelloAndroid.apk" />
+    <Exec Condition="'$(RunActivity)' != 'true'" Command="$(AdbTool) shell am instrument -w net.dot.HelloAndroid/net.dot.MonoRunner" />
```

Comment:
Remove this line - it's for headless launch, I guess we don't need it for samples. (and remove $(RunActivity))

Reply:
Thanks!